src/js/mep-player.js

```diff
@@ -1,3 +1,14 @@
+function isAfter(node, otherNode) {
```

Comment:
it's not obvious to me which one is supposed to be after the other.  i.e. is it true if `otherNode` is after `node`, or vice-versa?  that said, i can't come up with a great name to make it more clear.  `firstIsAfterSecond`?  maybe just a code comment.